packages/vm/tests/config.js

```diff
@@ -119,11 +119,17 @@ const SKIP_VM = [
  * @returns {String} Either an alias of the forkConfig param, or the forkConfig param itself
  */
 function getRequiredForkConfigAlias(forkConfig) {
+  // SpuriousDragon is named EIP158 (attention: misleading name)
+  // in the client-independent consensus test suite
+  if (String(forkConfig).match(/^spuriousDragon$/i)) {
```

Comment:
Are these aliases also integrated in common? (Not really related question to this PR)

Reply:
No, but I wouldn't integrate this in common, since this is only for the consensus tests and the internal names chosen there are not really correct partly and e.g. just refer to a single EIP of some HF and not the meta EIP from the HF itself. Wouldn't want to expose this via Common, this should rather be fixed at some point in the tests suite.

Reply:
You are right, implementing this in Common would lead to more confusion.